mobile-ui/src/main/java/com/playone/mobile/ui/injection/module/NetModule.kt

```diff
@@ -25,10 +25,8 @@ open class NetModule {
     }
 
     @Provides
-    internal fun provideChuckInterceptor(context: Context): ChuckInterceptor
-            = ChuckInterceptor(context)
+    internal fun provideChuckInterceptor(context: Context) = ChuckInterceptor(context)
 
     @Provides
-    internal fun provideStethoInterceptor(): StethoInterceptor
-            = StethoInterceptor()
+    internal fun provideStethoInterceptor() = StethoInterceptor()
```

Comment:
Memo: here should be careful since `Provider` cannot provide the same type without `Qualifier` for `Dagger`

Reply:
So what type to be returned should be precise.

Reply:
![jietu20180216-30141 pm](https://user-images.githubusercontent.com/5198104/36295330-50d5cbaa-132a-11e8-86e8-355d28e64879.png)

It should be okay here!